### DISCUSSION TITLE
OME blocks rtmp

### DISCUSSION THREAD

**Discussion by driwn** [OvenMediaLabs/OvenMediaEngine#1263] [Help]
using rtmp (port 1935) as provider in OME, but also I need rtmp in NodeJS app,
with nginx it works fine, (without anything doesn't works),
but with OME nodejs can't connect to the rtmp, 
how can I do this? with RTMPPush?

**Comment by mpisat** [comment]
maybe use a different port in your nodejs app for rtmp? like 1936?
